app/uk/gov/hmrc/merchandiseinbaggage/connectors/AddressLookupFrontendConnector.scala

```diff
@@ -23,7 +23,7 @@ import play.api.mvc.Call
 import uk.gov.hmrc.http.HttpReads.Implicits._
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
 import uk.gov.hmrc.merchandiseinbaggage.config.AddressLookupConfig
-import uk.gov.hmrc.merchandiseinbaggage.model.adresslookup.Address
+import uk.gov.hmrc.merchandiseinbaggage.model.api.addresslookup.Address
```

Comment:
we don't need separate package `addresslookup` as this package just has one class , but if you want to keep it that way, its fine

Reply:
it was like that but now looks a bit silly as you pointed out because I have Address and AddressLookup in the same file